scss/_utilities.scss

```diff
@@ -573,9 +578,12 @@ $utilities: map-merge(
       values: map-merge(
         $utilities-text-colors,
         (
-          "muted": $text-muted,
+          "muted": $text-muted, // deprecated
```

Comment:
Should we deprecate the `$text-muted`?

Reply:
I've kept the comment coming from Bootstrap. Everything is mentioned in the migration guide. It's not really really deprecated for now if I remember well.

Reply:
**[Bootstrap side]** Add a comment next to `$text-muted`